training/amd-bootc/Containerfile

```diff
@@ -1,6 +1,16 @@
+# Define the images to be used
+ARG INSTRUCTLAB_IMAGE="quay.io/ai-lab/instructlab-amd:latest"
```

Comment:
Shouldn't we add these pieces instead to the makefile for amd-bootc specifying the amd-instructlab image?